ios/engine/KMEI/KeymanEngine/Classes/Settings/InstalledLanguagesViewController.swift

```diff
@@ -165,49 +164,13 @@ class InstalledLanguagesViewController: UITableViewController, UIAlertViewDelega
     }
     
     cell.textLabel?.text = language.name
-    
-    if cell.accessoryType == .disclosureIndicator {
-      return
-    }
-    guard let cell = cell as? KeyboardNameTableViewCell else {
-      return
-    }
-    
-    let keyboardID = language.keyboards![0].id
-    
-    cell.accessoryType = .none
-    cell.isUserInteractionEnabled = true
-    cell.textLabel?.isEnabled = true
-    cell.detailTextLabel?.isEnabled = true
-    
-    let kbState = Manager.shared.stateForKeyboard(withID: keyboardID)
-    cell.setKeyboardState(kbState, selected: false, defaultAccessoryType: cell.accessoryType)
   }
   
   override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
     selectedSection = indexPath.section
     tableView.cellForRow(at: indexPath)?.isSelected = false
-    if tableView.cellForRow(at: indexPath)?.accessoryType == .disclosureIndicator {
-      let title = tableView.cellForRow(at: indexPath)?.textLabel?.text ?? ""
-      showLanguageSettingsView(title: title, languageIndex: indexPath.section)
-      return
-    }
-    
-    let language = languages[indexPath.section]
-    let keyboardIndex = 0;
-    let keyboard = language.keyboards![keyboardIndex]
-    
-    //TODO: launch real Language Settings controller here
-    let languageSettingsController = UIAlertController(title: "\(language.name): \(keyboard.name)",
-      message: "This  should be Language Settings",
-      preferredStyle: UIAlertControllerStyle.alert)
-    languageSettingsController.addAction(UIAlertAction(title: "Cancel",
-                                            style: UIAlertActionStyle.cancel,
-                                            handler: nil))
-//    languageSettingsController.addAction(UIAlertAction(title: "Download",
-//                                            style: UIAlertActionStyle.default,
-//                                            handler: {_ in self.downloadHandler(keyboardIndex)} ))
-    self.present(languageSettingsController, animated: true, completion: nil)
+    let title = tableView.cellForRow(at: indexPath)?.textLabel?.text ?? ""
+    showLanguageSettingsView(title: title, languageIndex: indexPath.section)
```

Comment:
There was a lot of code removed around here, and I'm not immediately catching the purpose.  Is it related to how the (i) info button is being handled now, or what's going on here?

Reply:
Updates aren't done.
Also, a dead giveaway to old code that knew it was incomplete is this: "This  should be Language Settings"

Reply:
@rfaramir Let us know when you are ready for re-review.

Reply:
I think it's ready.